src/container/BooksList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BooksList = ({ bookId, title, category }) => (
+  <table>
+    <th>BookId</th>
+    <th>Title</th>
+    <th>Category</th>
+    <Book bookId={bookId} title={title} category={category} />
```

Comment:
- [ ] [REQUIRED]  it should display `books list` in html table using `Book` component[ screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/BooksAsTable.png)